openmrs_concepts_1.9.9_20171118_schema_only.sql

```diff
@@ -0,0 +1,596 @@
+/*
+ Navicat Premium Data Transfer
+
+ Source Server         : OpenMRS1.9
+ Source Server Type    : MySQL
+ Source Server Version : 50509
+ Source Host           : localhost
+ Source Database       : liquibaserunner
+
+ Target Server Type    : MySQL
+ Target Server Version : 50509
+ File Encoding         : utf-8
+
+ Date: 11/19/2017 10:28:49 AM
+
+ This file is a subset of openmrs_concepts_1.9.9_20171118.sql. It contains the DDL parts only and all insert statements
+ were removed. Using this file when testing the build is much faster than testing with the full CIEL data set.
+
+ To use this file in a build change src/main/liquibase/liquibase-ciel-data.xml as follows:
+
+ Replace
+
+   <sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />
+
+ with
+
+   <sqlFile path="openmrs_concepts_1.9.9_20171118_schema_only.sql" encoding="UTF-8" />
+
+ Do *not* forget to revert the change after testing the build is finished.
```

Comment:
Is there a way we can make this work without having to change and then revert?

Reply:
Changing and reverting is only needed when loading the CIEL data takes forever and results in an OOM exception. With the DDL only you can test that all other things are working .